modules/database/src/main/kotlin/dev/proxyfox/database/records/system/SystemRecord.kt

```diff
@@ -43,7 +42,6 @@ open class SystemRecord : MongoRecord {
 
     @Serializable(InstantLongMillisecondSerializer::class)
     var timestamp: Instant = Clock.System.now()
-    var token: String = generateToken()
```

Comment:
Probably a thing to note: If token isn't bound per system, there should be some way of looking up token -> user or system and vice-versa.